build.gradle

```
@@ -30,6 +30,10 @@ dependencies {
     minecraft "com.mojang:minecraft:${project.minecraft_version}"
     mappings "net.fabricmc:yarn:${project.yarn_mappings}:v2"
     modCompile "net.fabricmc:fabric-loader:${project.loader_version}"
+
+    // Fabric Resource Loader for translations
+    modCompile "net.fabricmc.fabric-api:fabric-resource-loader-v0:${project.fabric_resource_loader_version}"
+    include "net.fabricmc.fabric-api:fabric-resource-loader-v0:${project.fabric_resource_loader_version}"
```

```suggestion
    modCompile fabricApi.module("fabric-resource-loader-v0", project.fabric_api_version)
    include fabricApi.module("fabric-resource-loader-v0", project.fabric_api_version)
```
Instead of using the hash for the Fabric API Resource Loader, `fabricApi.module` can be used which will automatically fetch the correct hash for the specified Fabric API version. This improves readability in the `gradle.properties` file and simplifies upgrading to newer versions of the Fabric API.